### PR TITLE
feat: Command prefix is now configurable via settings

### PR DESCRIPTION
**Pull request by ppardalj**
Many shard owners ask to set command prefix to other character rather than "`[`" (for example `"."`). Let's allow them to do it without changing code :)

### CODE REVIEW

File: Projects/UOContent/Commands/Handlers.cs [hunk 1]
```diff
@@ -21,7 +21,7 @@ public static class CommandHandlers
     {
         public static void Initialize()
         {
-            CommandSystem.Prefix = "[";
+            CommandSystem.Prefix = ServerConfiguration.GetSetting("command.prefix", "[");
```
Comment (kamronbatman) [inline Projects/UOContent/Commands/Handlers.cs:24]: Let's use `GetOrUpdateSetting` so we can set a default in the config file so people can modify it there.

**Review by kamronbatman — CHANGES_REQUESTED** [review]

**Comment by kamronbatman** [comment]
Great job, thank you!

File: Projects/UOContent/Commands/Handlers.cs (hunk 1 above)
Comment (kamronbatman) [inline Projects/UOContent/Commands/Handlers.cs:24]: Also a nitpick, but I am wondering if we should make the feature `commandsystem` so the config would be `commandsystem.prefix`.

Thoughts?

Comment (ppardalj) [reply Projects/UOContent/Commands/Handlers.cs:24]: Good idea, changing it! Thanks for feedback!

Comment (ppardalj) [reply Projects/UOContent/Commands/Handlers.cs:24]: Sounds good :)

**Review by kamronbatman — APPROVED** [review]

**Pull request merged**